Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Build Stage
```

Comment:
You don't need to do a git clone - `ADD .` is sufficient. Can we add a package stage here?